types/react-dom/index.d.ts

```diff
@@ -30,8 +30,8 @@ export const version: string;
 export const render: Renderer;
 export const hydrate: Renderer;
 
+export function flushSync(): void;
```

Comment:
I don't think we need this one. It's almost always a mistake to omit the callback and if you know what you're doing, passing a noop is not too much to ask.

A required callback is also matching docs: https://react.dev/reference/react-dom/flushSync

```suggestion
```

Reply:
Will revisit once https://github.com/reactjs/react.dev/pull/6730 is resolved.